front/components/vaults/CreateVaultModal.tsx

```diff
@@ -128,7 +130,12 @@ export function CreateVaultModal({
         const errorData = await res.json();
         throw new Error(errorData.error?.message || "Failed to create vault");
       }
-      return await res.json();
+      const r: PostVaultsResponseBody = await res.json();
+
+      // Invalidate the vaults list
+      await mutate(`/api/w/${owner.sId}/vaults`);
```

Comment:
We avoid using the global mutate and we prefer using the local one I guess the one from `useVaults` in this case.